unfoldingWord/translationCore
# Implement data persistence for module settings reducer

#### This pull request addresses:

This PR recovers data persistence for the scripture pane settings.


#### How to test this pull request:

Try adding a new language pane to the scripture pane and refresh. when refreshed it should persist the panes you added before refreshing.

Test with https://github.com/translationCoreApps/translationWords_Check_plugin/pull/64/files


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1046)
